bootstrap/bootstrap.go

```diff
@@ -291,8 +319,8 @@ func (bs *bootstrap) initialize() error {
 		//NOTE(longsleep): remove me - create keypair a random key pair.
 		sm := jwt.SigningMethodPS256
 		bs.signingMethod = sm
-		logger.WithField("alg", sm.Name).Warnf("missing --signing-private-key parameter, using random %d bit signing key", defaultSigningKeyBits)
-		signer, _ := rsa.GenerateKey(rand.Reader, defaultSigningKeyBits)
+		logger.WithField("alg", sm.Name).Warnf("missing --signing-private-key parameter, using random %d bit signing key", DefaultSigningKeyBits)
+		signer, _ := rsa.GenerateKey(rand.Reader, DefaultSigningKeyBits)
 		bs.signers[bs.signingKeyID] = signer
 	}
```

Comment:
Can this whole random key genertion stuff stay in the cmd package? Not particularly important but would also solve the problem with the duplications of the DefaultSigningKeyBits.